.gitlab-ci.yml

```diff
@@ -412,7 +412,51 @@ examples-docs:
       --path integration-tests
       -- cargo doc --manifest-path {} --document-private-items --verbose --no-deps
 
-#### stage:                        ink-waterfall
+#### stage:                        measurements
+
+contract-sizes:
+  stage:                           measurements
+  <<:                              *docker-env
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "push" && $CI_COMMIT_REF_NAME != "master"
+  variables:
+    # Fix for linking of `linkme` for `cargo contract build`: https://github.com/dtolnay/linkme/issues/49
```

Comment:
Why is `linkme` involved in our CI in the first place?

Reply:
It's to do with the events 2.0 metadata, there is a fix now so maybe we don't need that at all anymore, see https://github.com/paritytech/ink/pull/1921 and https://github.com/paritytech/cargo-contract/pull/1344